apps/docs/docs/api/modules/ts_rest_nest.md

```diff
@@ -0,0 +1,164 @@
+---
+id: "ts_rest_nest"
+title: "Module: @ts-rest/nest"
+sidebar_label: "@ts-rest/nest"
+sidebar_position: 0
+custom_edit_url: null
+---
+
+## Classes
+
+- [ApiRouteInterceptor](../classes/ts_rest_nest.ApiRouteInterceptor.md)
+
+## Type Aliases
+
+### ApiDecoratorShape
+
+Ƭ **ApiDecoratorShape**<`TRoute`\>: `Without`<{ `body`: `TRoute` extends `AppRouteMutation` ? `BodyWithoutFileIfMultiPart`<`TRoute`\> : `never` ; `params`: `PathParamsWithCustomValidators`<`TRoute`\> ; `query`: `ZodInferOrType`<`TRoute`[``"query"``]\>  }, `never`\>
+
+#### Type parameters
+
+| Name | Type |
+| :------ | :------ |
+| `TRoute` | extends `AppRoute` |
+
+#### Defined in
+
+[libs/ts-rest/nest/src/lib/api.decorator.ts:37](https://github.com/oliverbutler/tscont/blob/a69280b/libs/ts-rest/nest/src/lib/api.decorator.ts#L37)
```

Comment:
maybe this should point to `https://github.com/ts-rest` somehow?

Reply:
Yeah, you're right. It's annoying, every time I make a change in docs it outputs this link.

Which by itself is weird, its the OG path to ts-rest, and that name isn't _anywhere_ in the repo. So I'm not sure why it's doing that.